src/app/users/users-achievements/users-achievements-update.component.html

```diff
@@ -4,6 +4,7 @@
 </mat-toolbar>
 
 <div class="planet-users-achievements-update space-container">
+  <h2>{{user.firstName}} {{user.middleName}} {{user.lastName}}</h2>
   <mat-toolbar class="primary-color font-size-1">
     <span>{{user.name}}</span>
```

Comment:
The full name should replace the user name if profile is filled out.  See users-achievements.component.html

Reply:
Made suggested changes